project-proposals/stage-1/stage-1-questionnaire.md

```diff
@@ -1,8 +1,15 @@
 # Questionnaire for projects intending to join the OpenRail Association
 
-*As a project intending to join the OpenRail Association, please fill out this questionnaire and send it to the Technical Committee. See the [incubation process](../../incubation-process.md) for details about criteria and requirements for new projects and how it works to get new projects into the association.*
+\*As a project intending to join the OpenRail Association, please:
```

Comment:
The `*` were used to make the text italic. This doesn't work over multiple paragraphs. With the `\` you are escaping this, making it appear as a raw asterisk. This probably isn't intended.

Reply:
Yes indeed, my editor escaped the * for no reason at all. I'll fix it